### DISCUSSION TITLE
How  execute left join query  and order

### DISCUSSION THREAD

**Discussion by myfreax** [prisma/prisma#3320] [Q&A] [answered]
model description
```
model User{
  id
}
model Ticket {
  id
  title
  creator User
  userId
}

model Vote {
 id
 ticketId
 Userid
 action
}
```
Now, I want to query specif user ticket and relate ticket with user in vote model

equal SQL
```
'SELECT *  FROM "Ticket" LEFT JOIN "Vote" ON "Ticket"."userId" = "Vote"."userId"  WHERE "Ticket"."userId" = 1' order updateAt desc
```

**Comment by nikolasburk** [comment, marked as answer]
Hey @huangyanxiong01, have you tried this query?

```ts
await prisma.user.findOne({
  where: { id: 1 },
  include: {
    tickets: true
   }
})
```

I assume you have a field on `User` called `tickets`? Do you mind sharing the models including _all_ relation fields as well as the types and attributes? 🙂

**Reply by mxgs-p** [reply]
And ... where is order by 'update'?

**Reply by kasir-barati** [reply]
@nikolasburk I cannot see where you specified the join type - left join, right join, inner join, full join - in the code. Any answer or clarification for this vague API*?

\* With total respect to Prisma developers. I am just asking question.

@mxgs-p @myfreax did you know what kind of join would be performed against db when we are using Prisma?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
